portfolio/src/main/java/com/google/sps/servlets/UserServlet.java

```diff
@@ -0,0 +1,47 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.User;
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.LinkedHashMap;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(
+    name = "UserAPI",
+    description = "UserAPI: Login / Logout with UserService",
+    urlPatterns = "/userapi"
+)
+public class UserServlet extends HttpServlet {
+    
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        LinkedHashMap<String, String> Url_and_Bool = new LinkedHashMap<>();
+
+        UserService userService = UserServiceFactory.getUserService();
+        String thisUrl = request.getRequestURI();
+        String url;
+        String isLoggedIn;
+
+        if(!userService.isUserLoggedIn()) {
+            url = userService.createLoginURL("/index.html");
+            isLoggedIn = "false"; 
+         }
+        else {
```

Comment:
To conform with the styleguide, the else statement should be on the same line as the closing brace.